microsoft-edge/webview2/how-to/debug-visual-studio-code.md

```diff
@@ -109,8 +113,7 @@ You might need to attach the debugger to running WebView2 processes.  To do that
 "runtimeExecutable": "C:/path/to/your/webview2/myApp.exe",
 "env": {
    "Path": "%path%;e:/path/to/your/build/location; "
-},
-"useWebView": true
+}
```

Comment:
```suggestion
},
"useWebView": true
```
reverting this change, per [Comment](https://github.com/MicrosoftDocs/edge-developer/pull/3384/files#r1965590535) 
@mikosg @dhveerap